art-blocks-api/api-overview.md

```diff
@@ -269,6 +277,7 @@ Provides a broader set of the data that our front-end consumes — this includes
 | Ethereum | Staging (deprecated) | `https://ab-staging-goerli.hasura.app/v1/graphql`   |
 | Arbitrum | Production           | `https://ab-prod-arbitrum.hasura.app/v1/graphql`    |
 | Arbitrum | Staging              | `https://ab-staging-arbitrum.hasura.app/v1/graphql` |
+| Base | Mainnet              | `https://ab-prod-base.hasura.app/v1/graphql` |
```

Comment:
Missing staging link? Or maybe we don't have a staging env for rn

Reply:
There won't be one for L2s moving forward